package.json

```
@@ -8,13 +8,27 @@
   "type": "module",
   "exports": {
     ".": {
-      "types": "./dist/types/index.d.ts",
-      "require": "./dist/cjs/index.cjs",
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      },
+      "import": {
+        "types": "./dist/es/index.d.ts",
+        "default": "./dist/es/index.js"
+      },
+      "types": "./dist/es/index.d.ts",
       "default": "./dist/es/index.js"
```

You can shorten this since tsc will look for a `*d.ts|*.d.cts` with the same name automatically:
```suggestion
      "require": "./dist/cjs/index.cjs",
      "import": "./dist/es/index.js",
      "default": "./dist/es/index.js"
```